cmd/ocsp-updater/main.go

```diff
@@ -113,27 +120,33 @@ func findStaleResponses(cac rpc.CertificateAuthorityClient, dbMap *gorp.DbMap, o
 		log.Err(fmt.Sprintf("Error loading certificate status: %s", err))
 	} else {
 		log.Info(fmt.Sprintf("Processing OCSP Responses...\n"))
+		outerStart := time.Now()
+
 		for i, status := range certificateStatus {
-			log.Info(fmt.Sprintf("OCSP %d: %s", i, status.Serial))
+			log.Debug(fmt.Sprintf("OCSP %s: #%d", status.Serial, i))
+			innerStart := time.Now()
 
 			// Each response gets a transaction. To speed this up, we can batch
 			// transactions.
-			tx, err := dbMap.Begin()
+			tx, err := updater.dbMap.Begin()
 			if err != nil {
-				log.Err(fmt.Sprintf("Error starting transaction, aborting: %s", err))
+				log.Err(fmt.Sprintf("OCSP %s: Error starting transaction, aborting: %s", status.Serial, err))
 				tx.Rollback()
 				return err
 			}
 
-			if err := processResponse(cac, tx, status.Serial); err != nil {
-				log.Err(fmt.Sprintf("Could not process OCSP Response for %s: %s", status.Serial, err))
-				tx.Rollback()
-				return err
+			if err := updater.processResponse(tx, status.Serial); err != nil {
+				log.Err(fmt.Sprintf("OCSP %s: Could not process OCSP Response: %s", status.Serial, err))
+				continue
 			}
 
-			log.Info(fmt.Sprintf("OCSP %d: %s OK", i, status.Serial))
+			log.Info(fmt.Sprintf("OCSP %s: OK", status.Serial))
 			tx.Commit()
```

Comment:
Since you're in a loop here you may want to catch and check the `error` returned from `Commit()` in the same way as `Begin()`